cdap-docs/examples-manual/source/index.rst

```diff
@@ -4,41 +4,16 @@
 
 .. _examples-introduction-index:
 
-============================================
-CDAP Examples, How-To Guides, and Tutorials
-============================================
+==================
+CDAP How-To Guides
+==================
 
 
 .. |examples| replace:: **Examples:**
 .. _examples: examples/index.html
 
-- |examples|_ Included with the :ref:`CDAP Sandbox, <getting-started-index>` they range from a
-  simple introductory :ref:`Word Count <examples-word-count>` to more elaborate examples
-  such as the :ref:`Purchase application <examples-purchase>` that uses many of the CDAP
-  components.
-
-
 .. |guides| replace:: **How-To Guides:**
 .. _guides: how-to-guides/index.html
 
 - |guides|_ Designed to be completed in 15-30 minutes, these guides provide quick, hands-on
```

Comment:
seems that - since guide are the only thing that's left - we can move the contents of how-to-guide/index.rst here. Otherwise this page becomes a moot point, all it has is a pointer to another page?